### DISCUSSION TITLE
ZTD results of groops apear a large RMSE comparing with IGS-ZTD when using GPS L1/L5 and L2/L5

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#201] [Q&A]
Hello developers,
  When I used GPS L1/L5 or L2/L5 to compute ztd, I found there is a large bias compared to IGS-ZTD, even ztd results of L1/L2/L5 was worse than L1/L2 . It should be noted ambiguities are not resolved and wum ppp-ar products are used, and I sum 
**1.tropospheric zenith dry delay [m] (only from model)
2.tropospheric zenith wet delay [m] (model + delta estimate)
3.tropospheric dry gradient - north direction [m] (model + delta estimate, due to same mapping function)
4.tropospheric wet gradient - north direction [m] (only from model)
5.tropospheric dry gradient - east component [m] (model + delta estimate, due to same mapping function)
6.tropospheric wet gradient - east component [m] (only from model)**
of groops troposphere result file to get ztd.
  Config files of L1/L2 L1/L5 L2/L5 and L1/L2/L5 as well as list of station, RMSE result are in the attachment 
[attachment.zip](https://github.com/groops-devs/groops/files/11835576/attachment.zip)
  I did not know what happened exactly, hope to get some help about this issue.
Regards,
Yongzhao Fan

**Comment by FFFFFG-IGG** [comment]
> Hello developers, When I used GPS L1/L5 or L2/L5 to compute ztd, I found there is a large bias compared to IGS-ZTD, even ztd results of L1/L2/L5 was worse than L1/L2 . It should be noted ambiguities are not resolved and wum ppp-ar products are used, and I sum **1.tropospheric zenith dry delay [m] (only from model) 2.tropospheric zenith wet delay [m] (model + delta estimate) 3.tropospheric dry gradient - north direction [m] (model + delta estimate, due to same mapping function) 4.tropospheric wet gradient - north direction [m] (only from model) 5.tropospheric dry gradient - east component [m] (model + delta estimate, due to same mapping function) 6.tropospheric wet gradient - east component [m] (only from model)** of groops troposphere result file to get ztd. Config files of L1/L2 L1/L5 L2/L5 and L1/L2/L5 as well as list of station, RMSE result are in the attachment [attachment.zip](https://github.com/groops-devs/groops/files/11835576/attachment.zip) I did not know what happened exactly, hope to get some help about this issue. Regards, Yongzhao Fan

In fact, I've also tried ztd estimation with just double frequency of BDS3 or GALILEO, they also shows a large RMSE, It's weird.

**Comment by tmayerguerr** [comment]
Hi,
I see two possible problems:

1.  L5 phases in GPS should not be used because they are not stable in time (the phase bias is not constant but changes strongly per orbit).
2. Furthermore, I am not sure if you can simply add the gradients to the zentithdelay.

regards
Torsten

**Comment by FFFFFG-IGG** [comment]
> Hi, I see two possible problems:
> 
> 1. L5 phases in GPS should not be used because they are not stable in time (the phase bias is not constant but changes strongly per orbit).
> 2. Furthermore, I am not sure if you can simply add the gradients to the zentithdelay.
> 
> regards Torsten
Thanks Torsten,  
  However, Only float solution was applied and temporal bias well be merged into float ambiguities(I think).  In fact, I've also checked the position results, neither L1/L5 or L2/L5 performed well. 
regards,
Yongzhao Fan
